java/sage/VideoFrame.java

```diff
@@ -5650,9 +5651,42 @@ private int getMatchingLangIndex(String[] langOptions, String targetLang)
 
   private boolean selectDefaultSubpicLanguage()
   {
+    String defaultAudioLang = uiMgr.get("default_audio_language", "English"); 
+    MediaLangInfo mli = (MediaLangInfo) mediaLangMap.get(defaultAudioLang);	
+    
     String defaultLang = uiMgr.get("default_subpic_language", "");
-    if (defaultLang == null || defaultLang.length() == 0)
-      return true; // preferred no subtitles
+    
+    //Check for forced subtitles if there is not a defaul subtitle language set
+    if(defaultLang == null || defaultLang.length() == 0)
+    {
+      String [] subs = getDVDAvailableSubpictures();
+      boolean hasForcedSub = false;        
+
+      /*
+      * Loop thru all subtitle tracks to see if there is a forced subtitle track. 
+      * Clear the tracks that are not forced
+      */
+      for(int i = 0; i < subs.length; i++)
+      {
+        if(subs[i].toLowerCase().indexOf("[forced]") != -1)
+        {
+          hasForcedSub = true;
+        }
+        else
+        {
+          subs[i] = "";  
+        }
+      }
+        
+      if(hasForcedSub)
+      {
+        return performLanguageMatch(defaultAudioLang, subs, DVD_CONTROL_SUBTITLE_CHANGE, getDVDSubpicture());
```

Comment:
Don't just return here, if this returns false then you should fall-through to the normal subpicture selection.

Reply:
I have already checked above that there is no "default_subpic_language".  I believe we would be done if it was unable to find a match.  I believe we should be OK.  Am I wrong?

Reply:
Sorry, you're right. :)